soroban-env-host/src/host.rs

```diff
@@ -983,7 +1011,12 @@ impl Host {
                     let closure = AssertUnwindSafe(move || cfs.call(&func, self, args));
                     let res: Result<Option<RawVal>, PanicVal> = testutils::call_with_suppressed_panic_hook(closure);
                     match res {
-                        Ok(Some(rawval)) => Ok(rawval),
+                        Ok(Some(rawval)) => {
+                            // This is under testutils, so no need to check is_debug since it's
```

Comment:
I did this initially just as a sanity check in case the parameters passed into the diagnostics functions needed to be converted (potentially leading to a charge). It doesn't look like that applies at the moment though so I could stop doing this.

Reply:
It reduces the noise and mental burden on the reader so I would go ahead and remove the call site branches yeah.